deps/rabbit/src/rabbit_fifo.erl

```diff
@@ -1978,18 +2000,20 @@ get_next_msg(#?MODULE{returns = Returns0,
             Msg
     end.
 
-delivery_effect({CTag, CPid}, [{Idx, {MsgId, Header}}],
+delivery_effect({CTag, CPid}, [{MsgId, [Idx | Header]}],
```

Comment:
Use `?MSG(Idx, Header)` instead of `[Idx | Header]` ?

Reply:
cheers done